sb3_contrib/common/utils.py

```diff
@@ -67,3 +68,82 @@ def quantile_huber_loss(
     else:
         loss = loss.mean()
     return loss
+
+
+# TODO: write regression tests
+def cg_solver(Avp_fun: Callable[[th.Tensor], th.Tensor], b, max_iter=10, residual_tol=1e-10) -> th.Tensor:
+    """
+    Finds an approximate solution to a set of linear equations Ax = b
+
+    Source: https://github.com/ajlangley/trpo-pytorch/blob/master/conjugate_gradient.py
+
+    :param Avp_fun : callable
```

Comment:
minor: you don't need to specify the type in the docstring (as you do it in the model definition)
and there is no space after the parameter name: `:param b:` (not `:param b :` )